src/controllers/replies.controller/addReply/helpers/replyComment.js

```diff
@@ -0,0 +1,26 @@
+const { log } = require('../../../../config/logger');
+const { query: Hasura } = require('../../../../utils/hasura');
+const {
+  replyOnPostComment,
+  replyOnIdeaComment,
+  replyOnThoughtComment,
+} = require('../queries/mutations');
+
+async function replyComment(commentType, text, commentId, userId, parentReplyId) {
+  const mutationMap = {
```

Comment:
This mutation map can be moved out of function, and we can avoid object creation during every call